components/panel/color.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import PanelBody from './body';
+
+function PanelColor( { colorValue, title, initialOpen = false, ...props } ) {
```

Comment:
@mtias Do you mean for any colored panel to default to closed, or specifically text and background color on the paragraph block?